test/mongodb.test.js

```diff
@@ -38,7 +38,7 @@ describe('lazyConnect', function() {
     });
 
     ds.on('error', function(err) {
-      err.message.should.containEql('ECONNREFUSED');
+      err.message.should.match(/connect ECONNREFUSED/);
```

Comment:
I would personally go with `.match(/ECONNREFUSED/)`, but let's not bikeshed here too much. Pick whatever passes the tests and is not too specific.

No check will be entirely future proof, and the difference between the 70% chance and 90% chance of supporting newer mongodb versions is not worth wasting our time discussing it here.

That's my two cents, I'll refrain from more comments about this particular line of code.